survos/command-bundle/1.5/config/routes/survos_command.php

```diff
@@ -0,0 +1,12 @@
+<?php
+// config/routes.php
+
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return static function (RoutingConfigurator $routes): void {
+    $routes->import('@SurvosCommandBundle/config/routes.php')
+        ->prefix('/admin') // consider adding this route to the access_control key in security
```

Comment:
```suggestion
        ->prefix('/admin') // consider adding this path to the access_control key in security
```

Reply:
Btw, another option for you in the bundle is to, in your controller, always run a check for something like `ROLE_SURVOS_COMMAND`. Then tell the user to make sure their user's have this or put it in role_hierarchy. Then the controllers are locked down by default.

Reply:
interesting idea, I'll play with that for version 2 (which will support Symfony 7).